.gitignore

```diff
@@ -1,2 +1,3 @@
 sudoku
-__pycache__
\ No newline at end of file
+__pycache__
+.vscode
```

Comment:
why did you add that m8

Reply:
Because I am using vscode tool and I dont want to add autogenerated files...

Reply:
ok